README.md

```diff
@@ -22,29 +22,16 @@ A Quick Start and full API Reference can be found in [The Things Network Documen
 import org.eclipse.paho.client.mqttv3.MqttClient;
 import org.eclipse.paho.client.mqttv3.MqttException;
```

Comment:
Can't we import these from `org.thethingsnetwork.java.app.lib.Client`? Maven should take care of the dependency on paho so the user shouldn't need to be bothered right?